<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loggin</class>
 <widget class="QMainWindow" name="loggin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loggin</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_logging_loggin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_logging_user">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_logging_passwd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_loggin_user">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>user@host:port</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_loggin_passwd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loggin_color">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>67</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_loggin_pp">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_loggin_port">
       <property name="text">
        <string>22</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txt_logging_user</tabstop>
  <tabstop>txt_loggin_port</tabstop>
  <tabstop>txt_logging_passwd</tabstop>
  <tabstop>btn_logging_loggin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
